test/unit/boleto-object.spec.js

```diff
@@ -0,0 +1,142 @@
+const R = require('ramda')
```

Comment:
O Ramda foi adicionado apenas para fazer os testes? Se for esse o caso, seria bom tirar de dependencies e colocar em devDependencies. Caso contrário, ele será baixado junto com o pacote desnecessariamente.